admin_manual/release_notes.rst

```diff
@@ -11,6 +12,45 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.3_release_notes_label:
+
+Changes in 10.0.3
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.3 that need your attention:
+
+**The full ownCloud Server 10.0.3 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* It is now possible to directly upgrade from 8.2.11 to 10.0.3 in a single upgrade process.
+* Added occ command to list routes which can help administrators setting up network firewall rules.
+* 'occ upgrade' is now verbose by default.
+    * Administrators may need to adjust scripts for automated setup/upgrade procedures that rely on 'occ upgrade' outputs.
+
+* Reenabled medial search by default
+    * Enables partial search in sharing dialog autocompletion (e.g. a user wants to share with the user "Peter": Entering "pe" will find the user, entering "ter" will only find the user if the option is enabled)
+    * New default is set to enabled as there is no performance impact anymore due to the introduction of the user account table in ownCloud Server 10.0.1.
+    * Please check the setting. You need to disable it if the functionality is undesired.
+
+* All database columns that use the fileid have been changed to bigint (64-bits).
+    * For large instances it is therefore highly recommended to upgrade in order to avoid reaching limits.
+
+* **Removed "themes" folder TODO: ask Philipp if still possible to use themes folder**
```

Comment:
@phisch please clarify: if an instance still has a theme under the "themes" folder, would said theme still work when upgrading to 10.0.3 where the folder is removed ?

I'd assume that at least distro package upgrades would leave that folder alone @crrodriguez

Reply:
If you create a theme folder and insert a theme in there, it will still work. If it is an old theme, some paths might need to be adjusted so they follow the documentation. (old themes might have the directory inconsistency issue we fixed earlier this year).

Reply:
Hmm ok. Then we should probably remove this from the release notes as it doesn't require any admin attention if old themes continue to work.

Once we decide when to deprecate the switch, we'll need to announce it on other channels.

@pmaier1 can you remove this entry ?

Reply:
done